public/static/Espresso.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+ <title> marleigh - Espresso </title>
+ </head>
+
+<body>
+  <div class = "whole-page">
+<h1 class = "main" > Espresso in Seattle</h1>
+
+ <nav>
+  <a href="home">Home</a> |
+  <a href="about_me">About Me </a> |
+  <a href="blog">Blog</a> |
+  <a href="projects">Projects</a> |
+
+</nav>
+
+
+<p>
+  In some shape or form I've been a barista for over 14 years. The good part: I love coffee. The bad part: I love cofee.  This 14 years of experience, makes me a pretty freaking opinionated person where coffee and especially espresso are concerned.
+
+  What follows are some of the espresso experiences I've had in my first few weeks here in Seattle:
+
+
+  (ps - I have the right to change my mind on these. NOTHING is set in stone.)
+</p>
+<br>
+
+<h2 class = "subtitle" >Espresso Vivace</h2>
+<p> 
+  Vivace has been my second favorite. The bean and milk combine for a a very rich, full flavor. Delicious, but missing just a little something. I think it's personality. Maybe I just haven't encountered the right barista yet. 
+
+  They do have amazing almond croissants though. 
+</p>
+
+<h2 class = "subtitle">Cafe Vita</h2>
+<p> 
+  I love Vita. As does over half of the city.  Almost every restaurant is serving coffee roasted by Vita. But how is there espresso? In a word? Delectable. Hands down, if I'm looking for a cappucino or latte, Vita is where I go. I 'm quite fond of the Pacnor.  
+
+  For pure attitude and taste, Vita wins hands down. The music is always the best, the espress on strong and smooth.
+</p>
+
```

Comment:
I would flip these two rankings, because Vivace is CLEARY better than Cafe Vita ;) :heart: